test/evergreen.yml

```diff
@@ -161,6 +188,24 @@ functions:
 
           ${configure_env_vars|} $CMAKE $DEFINED_EVERGREEN_CONFIG_FLAGS ${posix_configure_flags|} -G "${cmake_generator|Ninja}" ./..
         fi
+  "python config check":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/cmake_build"
+      shell: bash
+      script: |
+        set -o errexit
+        set -o verbose
+        python_configured=$(grep PYTHON_EXECUTABLE: ./CMakeCache.txt | sed -e 's/.*=//')
+        if [ "${python_binary}" = '' -o "$(${python_binary} -V -V)" != "$($python_configured -V -V)" ]; then
+           echo "Python versions do not match.  Cmake version:"
+           echo "$python_configured"
+           $python_configured -V -V
+           echo "Variable used by evergreen:"
+           echo "${python_binary}"
+           ${python_binary} -V -V
```

Comment:
It looks like this line could be executed when python_binary is ''. Maybe this needs another if statement just in case?

Reply:
Fixed